mode/gfm/test.js

```diff
@@ -167,11 +137,6 @@
      "foo asfd:asdf bar");
 
   MT("notALink",
-     "[comment ```css]",
-     "[tag foo] {[property color]:[keyword black];}",
-     "[comment ```][link http://www.example.com/]");
```

Comment:
permanently removed this test coz it's not a valid CommonMark codeBlock termination... will need to update tokenizer later.